### DISCUSSION TITLE
How to get sponsor badge in github?

### DISCUSSION THREAD

**Discussion by SusheelThapa** [Schweinepriester/github-profile-achievements#628] [Q&A] [answered]
Sponsor badge in github profile

**Comment by Stratis-Dermanoutsos** [comment, marked as answer]
You have to find a github repository for an open source project and click the sponsor button on the top right corner.

<img width="1398" alt="Screenshot 2023-02-11 at 19 42 34" src="https://user-images.githubusercontent.com/38103639/218272931-349ad736-e8c2-43b4-92d9-9e986d8c954b.png">

Of course, you have to complete the contribution to get the achievement, which means spending money.
Thus, I suggest finding a project you would really be willing to support.

**Reply by fouadmxp** [reply]
💕

**Comment by mir161803** [comment]
To get the sponsor badge, you should sponsor open source project $. You could find some that maybe will interest you here https://github.com/sponsors 

Or do check some projects or people that you admire and hopefully they open for sponsor

btw the best way to get started is to try sponsoring the author of the dependencies that you use (https://github.com/sponsors)
![image](https://user-images.githubusercontent.com/37258290/218284054-b6faa02f-b250-437c-957f-82c3cef83bca.png)

**Comment by tauanesales** [comment]
You just have to publicly sponsor anyone for any value. Try sponsoring https://github.com/ArielMAJ for 1 dollar for example and you should get the badge immediately after.

**Comment by leonden** [comment]
You actually don't have to sponsor an open source project in particular. Any creator/profile is valid to sponsor if it has the sponsoring option active - duh. Hope that helps 😄

**Comment by DhrubajyotiBiswasfounderofDribll** [comment]
> Sponsor badge in github profile

<html>
<body>


# Public Sponsor
## >> Sponsoring open source work via [GitHub Sponsors](https://github.com/sponsors) <<
![image](https://github.com/Schweinepriester/github-profile-achievements/assets/108980710/0e7e0001-fccb-4d23-9ee7-64d218848592)



</body>
</html>

### To sponsor a repository, we need to click on the Sponsor button and we can sponsor any person or repository.
### Sponsoring also charges for your money and you need to contribute in open source public sposor repository.
![image](https://github.com/Schweinepriester/github-profile-achievements/assets/108980710/d187c4e3-8fbd-4155-af13-2788857363a5)

**Comment by RupeshDev18** [comment]
> Sponsor badge in github profile

To get the Sponsor badge on GitHub, you need to join GitHub Sponsors and sponsor an open source contributor. Here's how you can do it:
● Join GitHub Sponsors:
1. Navigate to GitHub Sponsors


2. If you are an organization owner, you have more than one eligible account. Click "Join the waitlist"

3. Follow the prompts to complete your contact information


4. Decide whether to receive sponsorship payouts via a bank account or through a fiscal host


● Sponsor an open source contributor:
1. Find an open source contributor you want to sponsor. You can search for them on GitHub Sponsors or directly visit their GitHub profile

2. On the contributor's GitHub profile, click the "Sponsor" button to start sponsoring them

3. Choose the sponsorship amount and frequency that works for you

4. Complete the sponsorship process by following the prompts

After sponsoring an open source contributor, you will receive a Sponsor badge on your GitHub profile
. This badge indicates that you have supported the open source community and can be displayed throughout GitHub.

**Comment by HarshDekate** [comment]
give  star to my repos i will give you badges

**Comment by the1Riddle** [comment]
just sponsor somebody and get the badge
